packages/perseus-editor/src/editor-page.tsx

```diff
@@ -25,6 +25,9 @@ import type {
 const {HUD} = components;
 
 type Props = {
+    // Additional templates that the host application would like to display
+    // within the Perseus Editor.
+    additionalTemplates?: Record<string, string>;
```

Comment:
Perfect!  I'd just suggest switching this to be a JSDoc string as that surfaces the comments better in tools like VSCode. 
```suggestion
    /** 
     * Additional templates that the host application would like to display
     * within the Perseus Editor.
     */
    additionalTemplates?: Record<string, string>;
```